fairmq/runFairMQDevice.h

```diff
@@ -73,6 +73,11 @@ int main(int argc, const char** argv)
 
         int result = runStateMachine(*device, config);
 
+        if (config.Count("version"))
+        {
+            pluginManager->ForEachPlugin([](fair::mq::Plugin& plugin){ std::cout << "plugin: " << plugin.GetName() << ": " << plugin.GetVersion() << std::endl; });
```

Comment:
```cpp
std::cout << "plugin: " << plugin << std::endl;
```
would give you the same string I also print in the debug log, it contains also maintainer and homepage. If the formatting is not good, you can change it in the overloaded `operator<<` in `Plugin.h`. I would prefer this unless you don't want maintainer and homepage in the `--version` output.